Update docker hub to separate start with docker-compose and start with docker image separatly (#15)
exo-docker/exo-community: README.md

## Quick start

### Easy way : with docker-compose

> eXo Platform Community edition is based on sub-components :

Reviewer: ```suggestion
Running the eXo Platform Community edition container requires additional components to run  :
```

- Database (mysql or psql)
- Elastic Search
- Mongodb

> Theses sub-components are not provided in the docker image exoplatform/exo-community.

Reviewer: ```suggestion
These components are not provided within the docker image exoplatform/exo-community.
```

> If you want to run all components in one action, we provide

Reviewer: ```suggestion
In order to run all components at once, we provide a Docker Compose file and 
```

> a [guide](https://docs.exoplatform.org/en/latest/GettingStartedeXoCommunity.html) based on a docker-compose file. This will allow

Reviewer: ```suggestion
a [quick start guide](https://docs.exoplatform.org/en/latest/GettingStartedeXoCommunity.html). It will allow
```

> you to start all components needed by the platform.

Reviewer: ```suggestion
you to start all components required to get up and running easily with the default configuration.
```

### Advanced way : with docker images

> If you prefer, you can run separatly each components by starting their own docker image. Then you have to use properties describe

Reviewer: ```suggestion
Alternatively, you may want to run each component separately with containers. To do this, you can use properties described
```

Reviewer: it may be useful to link to the container images required so user can review their own README and manually configure them.

Author: Modification proposed

The most basic way to start eXo Platform Community edition for *evaluation* purpose is to execute

```bash
docker network create -d bridge exo-network
docker run -v mongo_data:/data/db -p 27017:27017 --name mongo --network=exo-network mongo:4.4
docker run -e ES_JAVA_OPTS="-Xms2048m -Xmx2048m" -e node.name=exo -e cluster.name=exo -e cluster.initial_master_nodes=exo -e network.host=_site_ -v search_data:/usr/share/elasticsearch/data --name es --network=exo-network exoplatform/elasticsearch:2.0.3

> docker run -v exo_data:/srv/exo -p 8080:8080 -e EXO_ES_HOST=es --name exo --network=exo-network exoplatform/exo-community:6.3

Reviewer 2: I think it will be helpful to state that those commands will create and run containers, and that once the contaienrs are created successfully, they could be started with
`` docker start $CONTAINER_NAME ``

Author: added

When ready just go to <http://localhost:8080> and follow the instructions ;-)

Once containers successfully start, you can stop/start them with
```bash
docker stop $CONTAINER_NAME
docker start $CONTAINER_NAME

> ```bash

Reviewer 3: Please remove `bash` (I can't propose a change with ```)

Author: Fixed
